pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -393,6 +395,7 @@ func (c *Cluster) newMonConfig(monID int) *monConfig {
 		ResourceName: resourceName(daemonName),
 		DaemonName:   daemonName,
 		Port:         DefaultMsgr1Port,
+		PortName:     "tcp-msgr1",
```

Comment:
No need for this since it's set in service.go, right?

Reply:
You're right. I got a bit turned around, Sorry about the hassle.